infrastructure/terraform/web-server-service.tf

```diff
@@ -10,6 +10,10 @@ resource "random_string" "web_server_access_token" {
   override_special = "_-"
 }
 
+resource "random_string" "web_server_jwt_key" {
+  length = 66
```

Comment:
Is there a reason behind a length of 66?

Reply:
I copied it from `auth-service.tf`.  I don't know why I chose 66 there!